src/pykeen/training/slcwa.py

```diff
@@ -88,7 +88,11 @@ def _process_batch_static(
             raise AttributeError("Slicing is not possible for sLCWA training loops.")
 
         # split batch
-        positive_batch, negative_batch, positive_filter = batch
+        positive_batch = batch["positives"]
```

Comment:
I don't love using a dictionary over dataclasses, it makes it much harder to reason over the code and take advantage of the type checker / code hints

Reply:
ahh but I see the collate supports it natively, so maybe this is a reasonable tradeoff